### ISSUE TITLE
tabela z danymi?

### ISSUE THREAD

**Issue by flaminka** [flaminka/RiBigDataProjekt1#6]

**Comment by topolskib** [comment]
W sensie wyświetlić w oddzielnym tabie?

**Comment by flaminka** [comment]
Tak, ale trzeba pomyśleć jakie ciekawe dane - to co prof Biecek mówił, że i wykresy i dane, ale to takie rzucenie pomysłu. Może być razem z wykresem , pod np.

**Comment by topolskib** [comment]
Tak, bo surowych danych nie ma co wyświetlać - trzeba jakiś ciekawy wykres wymyślić

**Comment by ZosiaRog** [comment]
pomysł 1) czas spędzony przez użytkownika w Galerii od pierwszego do ostatniego urządzenia (boxplocik jakiś czy coś, np. z podziałem na dni tygodnia)
pomysł 2) typowa długość ścieżki, zależnie np.  od pory dnia, pierwszego urządzenia
pomysł 3) najbardziej/ najmniej popularne urządzenia (to już jest w wielkości kropki oznaczone)
